src/recruiter/recruiter.controller.ts

```diff
@@ -16,6 +17,11 @@ export class RecruiterController {
     return this.recruiterService.getForInterviewStudents();
   }
 
+  @Patch('/filter')
```

Comment:
I think it should be GET, because we aren't making any changes in DB. 
It should be GET with parameters.

Reply:
That endpoint is gonna work when recruiter click on button, so i think its not possible to use get. I thought about post/patch/put, but in this situation we have changing list on frontend. Maybe patch then?

Reply:
It'd work, but I don't know it is correct way, becasue like I say POST/PATCH is for updating data in DB. And in general you want to retrive data from DB but filtered. So it rather musth be GET.

Reply:
I changed it. I just thought it's impossible to send body in get method :D

Reply:
You can send query :)